pmlc/dialect/eltwise/ir/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# In order to include source and headers we have to do it this way.
+pmlc_add_mlir_dialect(eltwise)
+pmlc_add_mlir_interfaces(eltwise)
+add_mlir_doc(interfaces -gen-dialect-doc EltwiseInterfaces Eltwise/)
+add_mlir_doc(ops -gen-op-doc EltwiseOps Eltwise/)
+
+add_mlir_dialect_library(PMLCEltwise
+        dialect.cc
+        ops.cc
+
+        ADDITIONAL_HEADER_DIRS
+
+        DEPENDS
+                MLIROpAsmInterfaceIncGen
+                PMLCeltwiseDialectGen
```

Comment:
I'd say `PMLCEltwiseDialectGen`

Reply:
kk, involves some annoying string processing in the macros is all - uses the tablegen file name to populate that. Ideally the rule would be explicit and have a name and a list of tablegen files etc. I can at least make the name / td file separate

Reply:
Derp, that was how the old one worked, this is easy nm